packages/app/src/library/MainFooter/TokenPrice.tsx

```diff
@@ -11,13 +12,15 @@ export const TokenPrice = () => {
     },
   } = useNetwork()
   const { price, change } = useTokenPrices()
+  const fiat = getUserFiatCurrency() || 'USD'
```

Comment:
We could introduce a new `Currency` context and use it here / throughout the app.